src/pytest_postgresql/janitor.py

```diff
@@ -23,7 +24,8 @@ def __init__(
             port: str,
             db_name: str,
             version: Union[str, float, Version],
-            password: str = None
+            password: str = None,
+            isolation_level: int = ISOLATION_LEVEL_AUTOCOMMIT,
```

Comment:
remove the above new import and just use `psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT` psycopg2 is imported into the compat, not installed in the submodule :)

Reply:
done